### PR TITLE
Reword tree building problem and add example tree

### PR DESCRIPTION
**Pull request by ErikSchierboom**
The current description uses a rather hard to read description. I've tried to reword it to be more clear. Furthermore, I've added a sample tree to show how it can look like.

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
Whoops! Fixed that.

**Comment by robkeim** [comment]
Looks great, thanks @ErikSchierboom!

**Comment by robkeim** [comment]
@ErikSchierboom - I guess I should double check is there any special process around merging PRs in x-common? I've only ever merged in the F# track.

**Comment by ErikSchierboom** [comment]
@robkeim Not that I know of. Usually someone like @Insti also checks the change, which he already did. So I think it's cool.

**Comment by robkeim** [comment]
Ok sounds good, thanks for the quick confirmation @ErikSchierboom!

**Comment by ErikSchierboom** [comment]
It indeed *does* invert the meaning, but I figured the old one was incorrect. The new one also makes more sense to me. A parent ID should always be lower than a record ID.
